android/app/build.gradle

```diff
@@ -33,13 +33,18 @@ android {
 }
 
 dependencies {
-
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
```

Comment:
could you please install wallet core from GitHub package? don't commit binary aar directly into this repo